src/components/BoardUsers/BoardUsers.tsx

```diff
@@ -0,0 +1,32 @@
+import BoardUser from './BoardUser';
+import RestUserCounter from './RestUsersCounter';
+import './BoardUsers.scss';
+import Parse from 'parse';
+import { useSelector } from 'react-redux';
+import { ApplicationState } from 'types/store';
+
+export interface BoardUsersProps {
+  numOfUsersToShow: number;
+}
+
+const BoardUsers = ({ numOfUsersToShow }: BoardUsersProps) => {
+  const users = useSelector((state: ApplicationState) => state.users.all);
+  const currentUser = Parse.User.current();
+
+  const me = users.find(user => user.id === currentUser!.id);
+  const connectedUsers = users.filter(user => user.online);
+  const them = connectedUsers.filter(user => user.id !== currentUser!.id);
+  const usersToShow = them.splice(0, (them.length > numOfUsersToShow ? numOfUsersToShow - 1 : numOfUsersToShow));
+
+  return (
+    <div className="users">
+      <ul className="user-list">
```

Comment:
I think you can also combine the parent `div` and the `ul` into one element here. Also I would prefer the name `board-users` as the class name here for easier debugging

Reply:
done